gulpfile.js

```diff
@@ -40,25 +40,113 @@ var output_directory = path.join('openslides', 'static');
 
 // Catches all JavaScript files (excluded worker files) from all core apps and concats them to one
 // file js/openslides.js. In production mode the file is uglified.
-gulp.task('js', function () {
-    return gulp.src([
-            path.join('openslides', '*', 'static', 'js', '**', '*.js'),
-            '!' + path.join('openslides', 'core', 'static', 'js', 'core', 'pdf-worker.js'),
-        ])
+
+var paths = {
```

Comment:
I would prefer not to put all path definitions in one big container but put everything together that belongs together:

Example:
```javascript
// Catches all JavaScript files (excluded worker files) from all core apps and concats them to one
// file js/openslides.js. In production mode the file is uglified.
var js_path = [path.join(...), '!' + path.join(...)];
var js_task = function () {
    return gulp.src(js_path)
        .pipe(...);
};
var js_watch = function () {
    gulp.watch(js_path, js_task);
};


...

gulp.task('default', ...);

gulp.task('watch', function () {
    js_watch();
    ...
}
```

Reply:
I stuck to the examples and documentation from [https://github.com/gulpjs/gulp](the official repository). It seems that having all paths in a container is good practice and it feels very practical as well to have everything in one place.